### DISCUSSION TITLE
Clarification on Submission Expectations

### DISCUSSION THREAD

**Discussion by beckja** [ARCLab-MIT/STORM-AI-devkit-2025#12] [Q&A] [answered]
I have a few questions (and comments) about the submissions:
1. Is the submission propagation explicitly supposed to be at 10-minute intervals?  It appears that way from the baseline file, but it doesn't appear to be called out anywhere (or I just missed it).  Also, the baseline/persistence propagator starts at midnight prior to the initial state time, but the sat_density files appear to start at the ten-second mark just after the initial state time. The competition Submit page says the times are to match the provided ground-truth (sat_density) files, but the sat_density files will not be available for the private dataset, so some clearer guidance is needed.
2. Is the expectation really that the total run time will be less than 10 minutes as stated on the submission tutorial page in the wiki or 10 minutes per initial state file?  The total run time seems both highly dependent on the number of private test initial states (which I either missed or is not stated anywhere) and highly dependent on the use of propagation and type.  I just completed a run of the warm-up training set with the baseline persistence model (after fixing it to work with an initial time NOT in the omni2 file and to only set the f10.7 and ap indices once for each file rather than every propagation step).  This run (without multi-processing) took about 18.5 hours for 715 objects (of which 46 failed to propagate to completion).  So 10 minutes seems somewhat "out there" in terms of expectations. :)
3. Is the 6-minute delay at the end of the submission script still required for the EvalAI submission to run properly like it was last year?  If so, does this count against the 10-minute limit?

Regards,
Jeff

**Comment by SSHurtado** [comment, marked as answer]
Thank you for being so participative. Please find the answers below:

1. Yes, the submission is explicitly supposed to be at 10-minute intervals. I will check the documentation to make this more visible.

2. As the evaluation pipeline is still broken, we have been unable to determine the usual time it takes for participants' submissions to execute all the test cases. However, if we identify a bottleneck with the computational resources or notice that the majority of submissions take longer than the specified time, we will make the necessary adjustments and announce the change accordingly.

3. That was a past problem, but as far as we know, it still exists. Please keep it in mind for your submission files. Once more submissions are uploaded, we will verify whether the issue persists. If it does not, we will make an announcement.

Hope this helps!

**Reply by beckja** [reply]
"Participative" -- is that a nice word for "annoying"...?  :)

Thank you for your response.

**Reply by SSHurtado** [reply]
> "Participative" -- is that a nice word for "annoying"...?  :)
> 
> Thank you for your response.

Not at all! Your participation in the forum is greatly appreciated, as not many people are doing so, and I hope it helps encourage others to get involved.
